scripts/keystore-setup.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+echo $ENCODED_ITALIAAPP_RELEASE_KEYSTORE | base64 --decode >> ${HOME}/italia-app/android/app/italiaapp-release.keystore
```

Comment:
is it possible to store it to some temporary directory (e.g. `/tmp`) to avoid the chance to getting it persisted by circleci?

Reply:
Sure it is possible. We just need to also change the ITALIAAPP_RELEASE_STORE_FILE bar with the absolute path. Going to change it.